Weather/Weather/WViewController/ViewController.m

```diff
@@ -0,0 +1,158 @@
+//
+//  ViewController.m
+//  Weather
+//
+//  Created by Sangwook Wi on 5/2/17.
+//  Copyright © 2017 Sangwook Wi. All rights reserved.
+//
+
+#import "ViewController.h"
+#import "WModel.h"
+#import "WCenter.h"
+#import "WTableViewCell.h"
+
+@interface ViewController() {
+    WCenter *cInstance;
+    CGRect originalsize;
+    NSArray *dayModel;
```

Comment:
We try to define all our variables as properties so keep things consistent and to give them the proper attributes.